react/src/lib/components/WebvizViewToggleContainer/WebvizViewToggleContainer.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { useStore } from "../WebvizContentManager/WebvizContentManager";
+
+export type WebvizViewToggleContainerProps = {
+    children?: React.ReactNode;
+    showInViews?: string[];
+    notShowInViews?: string[];
+};
+
+export const WebvizViewToggleContainer: React.FC<WebvizViewToggleContainerProps> =
```

Comment:
`ViewVisibilityContainer`?

Reply:
`WebvizViewVisibilityContainer` as it uses the `WLF`-store or `ViewVisibilityContainer`?

Reply:
Since it's a component which is going to be used directly (not under the hood like the rest of WLF), I'd vote for `ViewVisibilityContainer`.

Reply:
Good point!